fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/DefaultResourcePermissionConfig.java

```diff
@@ -34,7 +35,7 @@
   @ConditionalOnProperty(
       value = "auth.permissions.source.account.resource.enabled",
       matchIfMissing = true)
-  @Order
+  @Order(Ordered.HIGHEST_PRECEDENCE + 100)
```

Comment:
at LOWEST_PRECEDENCE (the default) these are the last PermissionSources, which means that other sources (like ChaosMonkey) don't see the initial permission state from the resource configuration so they were always seeing an empty permission and doing nothing